src/mca/gds/ds12/gds_dstore.c

```diff
@@ -2304,10 +2304,6 @@ inline pmix_status_t _dstore_fetch(const char *nspace, pmix_rank_t rank, const c
         return rc;
     }
 
-    if (kvs) {
-        *kvs = NULL;
```

Comment:
For the reference: `	** CID 1415602:  Null pointer dereferences  (FORWARD_NULL)`